compiler/rustc_passes/src/stability.rs

```diff
@@ -55,6 +55,21 @@ impl InheritDeprecation {
     }
 }
 
+/// Whether to inherit const stability flags for nested items. In most cases, we do not want to
+/// inherit const stability, because nested items rarely need constness, and so
+/// should be treated as not const even if their parent is. However, trait methods
```

Comment:
```suggestion
/// inherit const stability: just because an enclosing `fn` is const-stable does not mean
/// all `extern` imports declared in it should be const-stable! However, trait methods
```